### ISSUE TITLE
"Add Event" form should inform users that they cannot upload images bigger than 2MB

### ISSUE THREAD

**Issue by OneHwang** [Shift2Bikes/shiftcal#175]
When users add images bigger than 2MB, the posting mechanism will break. Users should be warned. Moreover, when users attempt to upload large images, they should not see a message that they will receive an email with a link to manage the event.

**Comment by jordancrane** [comment]
Can we close this issue? It appears to have been resolved by commit https://github.com/Shift2Bikes/shiftcal/commit/40db7b00a033c011acba22dd5e7fd0ee557b1b4f.
![image](https://user-images.githubusercontent.com/15274119/40265656-2b48cc34-5af1-11e8-9c17-781551d246dd.png)

**Comment by OneHwang** [comment]
No, this issue is not ready to be closed. It needs to be addressed ASAP. It should be closed only if people who try to upload large images no longer see the message that they will receive the email with a link to manage the event. The commit you referred to helps, but people still upload images bigger than 2MB, not understanding that the post mechanism breaks.
